tests/test_tui_npm_package_artifact.py

```
@@ -55,11 +108,70 @@ def test_archive_round_trip_preserves_package_executables(tmp_path: Path) -> Non
+        launcher = packages / "cmux"
+        launcher.mkdir(parents=True, exist_ok=True)
+        (launcher / "package.json").write_text(
+            json.dumps(
+                {
+                    "name": "cmux",
+                    "version": VERSION,
+                    "bin": {"cmux": "bin/cmux.js"},
+                    "files": ["bin/cmux.js"],
+                    "optionalDependencies": {
+                        name: VERSION for name in TARGETS
+                    },
```

Ruff (0.16.1)

[warning] Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)